.opspec/build/op.yml

```diff
@@ -48,7 +48,12 @@ run:
               - -ce
               - |
                 apk add -U curl
-                curl -Os https://uploader.codecov.io/latest/alpine/codecov
+              
+                if [[ `uname -m` == "aarch64" ]]; then
```

Comment:
I tried to use $(...) syntax here but it didn't work. I'm not sure why but this syntax appears to work on local (apple silicon) and in CI (linux)